frontend/styles.css

```diff
@@ -4,31 +4,30 @@ body {
   overflow: hidden;
   width: 100vw;
   height: 100vh;
+  font-family: 'Roboto Condensed', sans-serif;
 }
 
 .container {
-  min-width: 130px;
-  margin-left: 1em;
-  text-align: center;
-}
-
-.time {
-  display: block;
-  font-size: 3em;
-  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
 }
 
 .date {
-  font-size: 1.2em;
-  font-weight: 300;
+  font-size: 5em;
```

Comment:
Does this scale with the container ?

Reply:
I changed it to make it scale following the viewport width (keeping a fallback to browser that does not support the feature)
```css
.date {
  font-size: 5em;
  font-size: 10vw;
  font-weight: 400;
}
```